Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cueball"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
We should bump a minor version up to `0.3.0` since the `Connection` API is changing.

Reply:
Doh! Updated, thanks.